src/components/parent-form/ParentCreatedPage.tsx

```diff
@@ -93,16 +96,21 @@ export const ParentCreatedPage: React.FC<ParentCreatedPageProps> = ({
                         </Stack>
                     </Box>
                 </Center>
+            ) : successful === "pending" ? (
+                <Spinner />
```

Comment:
Let's use our Loading component instead!
```suggestion
                <Loading />
```